bcda/cli.go

```diff
@@ -277,7 +318,7 @@ func importCCLFDirectory(filePath string) (success, failure, skipped int, err er
 
 		cclfvalidator, err := importCCLF0(cclf0)
 		if err != nil {
-			log.Errorf("Failed to import CCLF0 file: %v, Skipping CCLF8 file: %v and CCLF9 file: %v ", cclf0, cclf8, cclf9)
+			log.Errorf("failed to import CCLF0 file: %v, Skipping CCLF8 file: %v and CCLF9 file: %v ", cclf0, cclf8, cclf9)
```

Comment:
Not important to change this now, but wanted to highlight that these `log.Error` calls are different from the rule of lowercase messages for errors. It's writing directly to the error level log, so this message wouldn't get wrapped as it bubbles up. It can be a complete sentence (or whatever we want 😄 )